pkg/storagesvc/stowClient.go

```diff
@@ -83,22 +109,14 @@ func MakeStowClient(logger *zap.Logger, storage Storage) (*StowClient, error) {
 		return nil, err
 	}
 	stowClient.location = loc
-
 	con, err := loc.CreateContainer(config.storage.getContainerName())
```

Comment:
Can we define one more wrapper function here?


```
func getOrCreateContainer(loc stow.Location, containerName string, cursor string) (stow.Container, error) {
        con, err := loc.CreateContainer(config.storage.getContainerName())
        if err != nil && (os.IsExist(err) || strings.Contains(err.Error(), "BucketAlreadyOwnedByYou")) {
                conn, err = getContainer(loc, config.storage.getContainerName(), stow.CursorStart)
        }
        return conn, err
}
```
This reduces errors checks and overall code.